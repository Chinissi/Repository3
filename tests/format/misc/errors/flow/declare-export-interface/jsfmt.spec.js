run_spec(
  {
    importMeta: import.meta,
    snippets: ["declare export interface A {}"],
  },
  ["flow", "babel-flow"]
);
