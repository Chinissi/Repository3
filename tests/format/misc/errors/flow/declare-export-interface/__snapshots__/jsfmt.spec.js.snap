// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snippet: #0 [babel-flow] format 1`] = `
"\`declare export interface\` is not supported. Use \`export interface\` instead. (1:16)
> 1 | declare export interface A {}
    |                ^"
`;

exports[`snippet: #0 [flow] format 1`] = `
"\`declare export interface\` is not supported. Use \`export interface\` instead. (1:16)
> 1 | declare export interface A {}
    |                ^^^^^^^^^"
`;
